tfe/resource_tfe_registry_module.go

```diff
@@ -85,6 +87,18 @@ func resourceTFERegistryModuleCreate(d *schema.ResourceData, meta interface{}) e
 			"Error creating registry module from repository %s: %v", *options.VCSRepo.Identifier, err)
 	}
 
+	doneCh := make(chan bool)
+	ctx, cancel := context.WithTimeout(ctx, time.Duration(5)*time.Minute)
+	defer cancel()
+
+	go checkModuleExists(ctx, tfeClient, registryModule.Organization.Name, registryModule.Name, registryModule.Provider, doneCh)
+	select {
+	case <-ctx.Done():
+		return fmt.Errorf("timeout while waiting for module to finish importing in registry.")
```

Comment:
nit: almost all of the errors are capitalized currently so we should capitalize this one for consistency
```suggestion
		return fmt.Errorf("Timeout while waiting for module to finish importing in registry")
```